src/include/execution/compiler/operator/static_aggregation_translator.h

```diff
@@ -5,6 +5,7 @@
 #include "execution/compiler/operator/operator_translator.h"
 #include "execution/compiler/pipeline.h"
 #include "execution/compiler/pipeline_driver.h"
+#include "planner/plannodes/aggregate_plan_node.h"
```

Comment:
Could you forward declare AggregateExpression instead? This helps a lot with compile times, because including the plan nodes tends to bring in a ton of code. (If you're interested, I have a script for counting the effects of an include.)

Reply:
Ah, rookie mistake. Thanks for the catch!